Auth state changed: SIGNED_IN 7bad0693-9053-423f-a2f0-a61c20b9cbf4
@supabase_supabase-js.js?v=4d4f5f04:3900 
            
            
           GET https://yekzyvdjjozhhatdefsq.supabase.co/rest/v1/conversation_participants?select=conversation_id&user_id=eq.7bad0693-9053-423f-a2f0-a61c20b9cbf4 404 (Not Found)
(anonymous) @ @supabase_supabase-js.js?v=4d4f5f04:3900
(anonymous) @ @supabase_supabase-js.js?v=4d4f5f04:3921
fulfilled @ @supabase_supabase-js.js?v=4d4f5f04:3873
Promise.then
step @ @supabase_supabase-js.js?v=4d4f5f04:3886
(anonymous) @ @supabase_supabase-js.js?v=4d4f5f04:3888
__awaiter6 @ @supabase_supabase-js.js?v=4d4f5f04:3870
(anonymous) @ @supabase_supabase-js.js?v=4d4f5f04:3911
then @ @supabase_supabase-js.js?v=4d4f5f04:89Understand this errorAI
Community.tsx:252 Error starting chat: {code: '42P01', details: null, hint: null, message: 'relation "public.conversation_participants" does not exist'}
handleStartChat @ Community.tsx:252
await in handleStartChat
onClick @ Community.tsx:388
handleClick @ react-router-dom.js?v=c660869e:5342
callCallback2 @ chunk-W6L2VRDA.js?v=0c0a4a81:3674
invokeGuardedCallbackDev @ chunk-W6L2VRDA.js?v=0c0a4a81:3699
invokeGuardedCallback @ chunk-W6L2VRDA.js?v=0c0a4a81:3733
invokeGuardedCallbackAndCatchFirstError @ chunk-W6L2VRDA.js?v=0c0a4a81:3736
executeDispatch @ chunk-W6L2VRDA.js?v=0c0a4a81:7014
processDispatchQueueItemsInOrder @ chunk-W6L2VRDA.js?v=0c0a4a81:7034
processDispatchQueue @ chunk-W6L2VRDA.js?v=0c0a4a81:7043
dispatchEventsForPlugins @ chunk-W6L2VRDA.js?v=0c0a4a81:7051
(anonymous) @ chunk-W6L2VRDA.js?v=0c0a4a81:7174
batchedUpdates$1 @ chunk-W6L2VRDA.js?v=0c0a4a81:18913
batchedUpdates @ chunk-W6L2VRDA.js?v=0c0a4a81:3579
dispatchEventForPluginEventSystem @ chunk-W6L2VRDA.js?v=0c0a4a81:7173
dispatchEventWithEnableCapturePhaseSelectiveHydrationWithoutDiscreteEventReplay @ chunk-W6L2VRDA.js?v=0c0a4a81:5478
dispatchEvent @ chunk-W6L2VRDA.js?v=0c0a4a81:5472
dispatchDiscreteEvent @ chunk-W6L2VRDA.js?v=0c0a4a81:5449Understand this errorAI
@supabase_supabase-js.js?v=4d4f5f04:3900 
            
            
           GET https://yekzyvdjjozhhatdefsq.supabase.co/rest/v1/conversation_participants?select=conversation_id&user_id=eq.7bad0693-9053-423f-a2f0-a61c20b9cbf4 404 (Not Found)
(anonymous) @ @supabase_supabase-js.js?v=4d4f5f04:3900
(anonymous) @ @supabase_supabase-js.js?v=4d4f5f04:3921
fulfilled @ @supabase_supabase-js.js?v=4d4f5f04:3873
Promise.then
step @ @supabase_supabase-js.js?v=4d4f5f04:3886
(anonymous) @ @supabase_supabase-js.js?v=4d4f5f04:3888
__awaiter6 @ @supabase_supabase-js.js?v=4d4f5f04:3870
(anonymous) @ @supabase_supabase-js.js?v=4d4f5f04:3911
then @ @supabase_supabase-js.js?v=4d4f5f04:89Understand this errorAI
Chat.tsx:26 Error loading conversations: {code: '42P01', details: null, hint: null, message: 'relation "public.conversation_participants" does not exist'}